OrangeTree_project_lunch.rb

```diff
@@ -0,0 +1,88 @@
+
+# Orange Tree class
+  class OrangeTree
+    attr_reader :height, :age, :orange_count, :max_age, :dead
+
+    def initialize
+      @height = 0
+      @age = 0
+      @orange_count = 0
+      @max_age = 100
+      @dead = false
+    end
+
+    # When one year passes, increases age and height
+    def one_year_passes
+      @orange_count = 0
+      @age += 1
+      @height += 0.75
+
+      # trees less than 3 years of age don't grow oranges
+      if @age < 3
+        @orange_count = 0
+
+      # once max age is reached, tree dies
+      elsif @age >= @max_age
+        @dead == true
+        puts "Yo tree is ded."
+
+      # after 3 years, orange count increases
+      else
+        @orange_count = @age * 2
+      end
+
+    end
+
+    # outputs count of oranges at current age
+    def count_the_oranges
```

Comment:
The assignment says that this should return the orange_count (as opposed to ```puts```, which prints it out, but returns ```nil```, a special value that means "nothing"). 

The difference is that, if you return the orange count, some other code (say, in the OrangeGrove) can call this method and then use the value to do some other calculation.